src/cloneDeep.js

```diff
@@ -1,5 +1,13 @@
 'use strict'
 
-const cloneDeep = (obj) => JSON.parse(JSON.stringify(obj))
+const isObject = require('./isObject')
+
+const cloneDeep = (obj) => {
+  if (!isObject(obj)) {
```

Comment:
To keep consistency with lodash
we should add: if (obj instanceof Function) return {}